lib/Conch/Controller/User.pm

```diff
@@ -95,7 +95,7 @@ sub set_setting ($c) {
 
     my $key = $c->stash('key');
     my $value = $input->{$key};
-    return $c->status(400, { error => "Setting key in request object must match name in the URL ('$key')" }) unless $value;
+    return $c->status(400, { error => 'Setting key in request object must match name in the URL (\''.$key.'\')' }) if not $value;
```

Comment:
I don't understand the intent of both of these changes, I've already brought up the quote change, but the `if not` seems a little pointless in this instance. Is there a performance difference?

Reply:
`unless` is a lot harder to read because one needs to push a stack frame "this entire expression is being negated". IMO.  I avoid it in new code and replace it when I can.